logging-infrastructure/README.md

```diff
@@ -0,0 +1,62 @@
+# Logging infrastructure
+
+As a Giant Swarm engineer, I want to be able to access a history of logs for components managing our platform in order to both investigate ongoing operational issues and provide details for incident reports.
+
+## Context
+
+Atlas started working on a logging infrastructure implementation using [Loki](https://grafana.com/docs/loki/latest/).
+So far we worked on a POC towards having a centralized setup, where a central Loki instance hosted by GiantSwarm ingests logs from our installations.
+There were some recent discussions where we learned about other requirements leaning towards a distributed setup, here are the pro and cons details of those discussions.
+
+### Distributed setup
+
+What we call distributed setup, is when there is 1 Loki instance hosted on each installation.
+So logs are stored and accessible on a per installation basis.
+
+#### Pros
+
+- Satisfy customer requirement towards data isolation (i.e. legal, or data privacy requirements)
+- More resilient setup towards failure
+- Keep our Platform as a Product model (self-sufficient installation can keep running alone)
```

Comment:
if customers are leaving, they get to keep their Loki. We could probably work on a "migration" path to get the centralized logs in an installation Loki for them if they leave but that would simplify it

Reply:
That's a workaround, but it does not change the point that if we introduce a centralized Loki we would move away from having everything in the installation.

Reply:
Yes but:
1: self sufficiency makes sense when they leave, not when they use the product, around which we can build a workaround
2: It depends where you draw the line with the platform. Some see it as the installation level, I prefer to see the platorm at the customer level, but then, why do we have multiple installations for the same customer. This is mostly coming from a customer centric approach (at least that's how I see it)

Reply:
I get your point, maybe I mixed too many concept in this one. Maybe I just remove this point as we already have the opposite argument in centralized setup as `Customer does not own their data (i.e. lose logs access when leaving GiantSwarm)`

Reply:
removed